packages/core/src/lib/notify.svelte

```diff
@@ -0,0 +1,127 @@
+<svelte:options immutable />
+
+<script lang="ts">
+type Variants = 'danger' | 'warning' | 'success' | 'info';
+
+import cx from 'classnames';
+import { createEventDispatcher } from 'svelte';
+import { Button, Icon } from '$lib';
+
+export let title = '';
+export let message = '';
+export let variant: Variants = 'info';
+export let progress = 1;
+export let exitable = false;
+
+$: isExitable = exitable;
```

Comment:
I think you shouldn't need isExitable at all now and you can just use exitable.

Reply:
removed and am now just using exitable